pkg/svcat/service-catalog/instance.go

```diff
@@ -37,7 +38,7 @@ const (
 func (sdk *SDK) RetrieveInstances(ns, classFilter, planFilter string) (*v1beta1.ServiceInstanceList, error) {
 	instances, err := sdk.ServiceCatalog().ServiceInstances(ns).List(v1.ListOptions{})
 	if err != nil {
-		return nil, fmt.Errorf("unable to list instances in %s (%s)", ns, err)
+		return nil, errors.Wrapf(err, "unable to list instances in %s (%s)", ns, err)
```

Comment:
when using errors.Wrapf, the error message is concatenated with the format text, so you don't need the final (%s), and err param.